game/scripts/vscripts/libraries/timers.lua

```diff
@@ -201,23 +204,24 @@ function Timers:CreateTimer(name, args, context)
 			context = args
 		end
 		args = {callback = name}
-		name = DoUniqueString("timer")
+		name = args --backwards compatibility
```

Comment:
Since we don't really use `name` anymore, maybe it can be removed completely and just `return args` at the end?

And since timers aren't named now, maybe `name` argument can be renamed to something like `arg1`?